src/views/Home/index.tsx

```diff
@@ -113,9 +211,12 @@ function Home(props: ViewProps) {
                                     {_ts('home', 'setupNewProjectButtonLabel')}
                                 </ButtonLikeLink>
                             </header>
-                            <RecentProjects
-                                className={styles.recentProjects}
-                                selectedProject={selectedProject}
+                            <ListView
+                                data={finalRecentProjects}
+                                rendererParams={recentProjectsRendererParams}
+                                renderer={ProjectItem}
+                                keySelector={recentProjectKeySelector}
+                                pending={projectDetailsPending || projectStatsPending}
```

Comment:
why are we using ListView to render a singular recent project?

Reply:
We'll later use the recent projects API to show 3 projects.